CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Bento Releases
 
+## Version 15.3.3
```

Comment:
This change adds a new feature so it should be a minor version bump (we go by [semver](https://semver.org/)) ie. `15.4.0`